src/graphics/device.js

```diff
@@ -1839,6 +1858,46 @@ pc.extend(pc, function () {
 
         /**
          * @function
+         * @name pc.GraphicsDevice#setTransformFeedbackBuffer
```

Comment:
This is called by engine it self, and not intended for a user, so consider adding `@private` after `@name` so to avoid it getting into documentation.